dev_docs/HOW_TO.md

```diff
@@ -11,14 +11,17 @@ Adding a new intake field requires changes to both apm-data and apm-server repos
    - RUM v3: [input/elasticapm/internal/modeldecoder/rumv3/model.go](/input/elasticapm/internal/modeldecoder/rumv3/model.go)
 2. Run `make generate` to generate the corresponding `model_generated.go` and JSON specification from the modified `model.go` in step 1.
 3. Run `make update-licenses` to add license header to `model_generated.go` generated in step 2.
-4. Add the new field to the corresponding file in `model/`, e.g. `span.go`.
-   1. Add the field to the struct.
-   2. Modify the `fields` function to include the field.
+4. Add the new field to the corresponding file in `model/proto/`, e.g. `span.proto`.
+   1. Run `make generate-modelpb` to generate the corresponding `span.pb.go`.
```

Comment:
Should we also mention that `span_vtproto.pb.go` would also be generated, and we prefer to use the methods from that file when needing to marshal and unmarshal with some brief introduction(as far as I know that was the case)?

Reply:
> Should we also mention that span_vtproto.pb.go would also be generated,

Good catch! :+1: 

>  and we prefer to use the methods from that file when needing to marshal and unmarshal with some brief introduction(as far as I know that was the case)?

I planned to add that as comments/doc to the methods to make it more visible when using the library. Not sure if that approach makes sense, WDYT ?

Reply:
IMO this is worth both - adding comments and describing the difference and when to use what in the dev docs.

Reply:
I'm with @simitt. As well as the comments on the methods, having an overview or a brief explanation about why we have `{entity}__vtproto.pb.go` files and why we choose to introduce them could help users to utilize them. (Based on my experience as I didn't know about this before Marc told me about it :) )

Reply:
Sorry about that! I've added more sections :bow: 

Let me know what you think!